src/main/resources/log4j.properties

```diff
@@ -18,5 +18,6 @@ log4j.category.org.hibernate.validator.internal.util.Version=WARN
 log4j.category.org.springframework.boot.actuate.autoconfigure.CrshAutoConfiguration=WARN
 log4j.category.org.springframework.boot.actuate.endpoint.jmx=WARN
 log4j.category.org.thymeleaf=WARN
-# Do not print consumer configuration on each consumer creation
-log4j.category.org.apache.kafka.clients.consumer.ConsumerConfig=WARN
\ No newline at end of file
+log4j.category.org.zalando.nakadi=DEBUG
+log4j.category.org.zalando.nakadi.config=INFO
+log4j.category.org.apache.kafka=WARN
```

Comment:
@adyach because we have WARN level for root kafka package anyway "org.apache.kafka"